### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by armanism24**
Updated NGINX HTTPS redirection.

### CODE REVIEW

**Comment by armanism24** [comment]
@googlebot I signed it!

File: tutorials/https-load-balancing-nginx/index.md [hunk 1]
```diff
@@ -352,7 +352,8 @@ balancer:
 
         server {
           listen 80;
-          return 301 https://$host$request_uri;
+           if ($scheme = "http") {
```
Comment (ToddKopriva) [inline tutorials/https-load-balancing-nginx/index.md:355]: unmatched opening bracket

File: tutorials/https-load-balancing-nginx/index.md [hunk 2]
```diff
@@ -352,7 +352,8 @@ balancer:
...
+           return 301 https://$host$request_uri;
         }
```
Comment (armanism24) [inline tutorials/https-load-balancing-nginx/index.md:357]: here we have the closing bracket. reference - [NGINX](https://www.nginx.com/resources/wiki/start/topics/examples/SSL-Offloader/#redirector-server)
here is the [screenshot](https://i.ibb.co/4jjcL9s/Screenshot-2021-07-05-at-10-31-15-PM.png)

File: tutorials/https-load-balancing-nginx/index.md (hunk 1 above)
Comment (armanism24) [reply tutorials/https-load-balancing-nginx/index.md:355]: here we have the closing bracket. reference - [NGINX](https://www.nginx.com/resources/wiki/start/topics/examples/SSL-Offloader/#redirector-server)
here is the [screenshot](https://i.ibb.co/4jjcL9s/Screenshot-2021-07-05-at-10-31-15-PM.png)

**Comment by armanism24** [comment]
Hi @ToddKopriva ,
Any update on this?

File: tutorials/https-load-balancing-nginx/index.md (hunk 1 above)
Comment (ToddKopriva) [reply tutorials/https-load-balancing-nginx/index.md:355]: In your modification, the `server` block's opening bracket is now left unmatched.

Comment (armanism24) [reply tutorials/https-load-balancing-nginx/index.md:355]: My bad, I missed the closing brackets for the ``server`` block, now fixed.

Comment (ToddKopriva) [reply tutorials/https-load-balancing-nginx/index.md:355]: Thanks for the fix.

**Review by ToddKopriva — APPROVED** [review]

**Comment by ToddKopriva** [comment]
Thanks for the update, @armanism24 .

**Pull request merged**